syntax = "proto3";

package shaggy;

message STFT {
  optional int32 frame_number = 1;
  optional int32 num_times_0 = 2;
  optional int32 num_fft = 3;
  optional int32 sample_rate = 4;
  optional int32 num_channel_2 = 5;
  optional bytes stft_samples = 6;
}
